src/main/java/org/jboss/wsf/spi/security/SecurityDomainContext.java

```diff
@@ -36,6 +38,7 @@
 public interface SecurityDomainContext
 {
    public String getSecurityDomain();
+   public SecurityDomain getSecurityDomainObject();
```

Comment:
Since this is public interface, let me know if I should rename this method to getSecurityDomain and delete the above method, or rename the above method to something else.